radar/data_management/pubsub.yaml

```diff
@@ -0,0 +1,32 @@
+name: Cloud Pub/Sub
+blip:
+  - date: 2019-06-13
+    ring: ADOPT
+description: |
+  [Cloud Pub/Sub](https://cloud.google.com/pubsub/architecture) is a fully-managed real-time messaging service.
+  That allows you to send and receive messages between independent applications.
+
+  By providing many-to-many, asynchronous messaging that decouples senders and receivers,
+  it allows for secure and highly available communication among independently written applications.
+rationale: |
+  Pub/Sub brings the flexibility and reliability of enterprise messaging to Extenda Retail.
+  It will provide us with a scalable, durable event ingestion and delivery system.
+  This will make sure that we have a good foundation for building modern streaming messaging pipelines.
+  For developers Cloud Pub/Sub delivers low-latency, durable messaging that can quickly be integrated with other systems
+  hosted on the Google Cloud Platform and external sources.
+related:
+  - data_management/kafka.yaml
+  - data_management/msmq.yaml
+license:
```

Comment:
Should we make this dual-licensed and also add `open-source` with Apache-2.0 for the client libraries?

Reply:
I think the client changes should be reflected on the clients if we use them.